src/style.css

```
@@ -1,3 +1,67 @@
-body {
+:root {
+  --current-text: #000;
+  --brend-color: #00acdc;
```

variable name mistake
```suggestion
  --brand-color: #00acdc;
```